ast2vec/repo2/base.py

```diff
@@ -166,15 +182,39 @@ def convert_uasts(self, file_uast_generator):
 
     def _bblfsh_parse(self, thread_index, filepath, language):
         from grpc import RpcError
+        status = "ERR"
+        start_time = time()
+        lines = lines_count(filepath, self._log)
         try:
-            return self._bblfsh[thread_index].parse(
+            res = self._bblfsh[thread_index].parse(
                 filepath, language=language, timeout=self._timeout)
+            status = "OK"
+            return res
         except DecodeError as e:
             msg = "bblfsh: DecodeError on %s: %s\nYour protobuf may be <= v3.3.2 " \
                   "and you hit https://github.com/bblfsh/server/issues/59#issuecomment-318125752"
             self._log.error(msg, e)
         except RpcError as e:
             self._log.error("bblfsh: RpcError on %s: %s", filepath, e)
+            self._log.warning(msg)
+            raise e from None
+        finally:
+            self._write_debug_logs(thread_index, filepath, status, lines, start_time)
+
+    def _write_debug_logs(self, thread_index, filepath, status, lines, start_time):
+        time_spent = time() - start_time
+        if not isnan(lines):
+            self._lines += lines
+            self._time += time_spent
+        if self._log.isEnabledFor(logging.DEBUG):
```

Comment:
Please check an environment variable instead. I think that if you do not define `AST2VEC_PARSE_LOG` then we should not write anything. This implies there is no `DEFAULT_PARSE_LOG_FILENAME `